### ISSUE TITLE
Improve pypesa directory structure

### ISSUE THREAD

**Issue by SentinelWarren** [Kalebu/pypesa#6]
Greetings @Kalebu , so this just a suggestion on the package structure improvement. Suggestion list goes as follows;

- Use `__init__.py` to only Import key classes | methods from modules directly into the package namespace. i.e the folder structure could then be;
```
pypesa/
__init__.py
mpesa.py
mpesa_execptions.py
service_urls.py
```
- `__init__.py` could also include other important parameters like, version, author, release date, doc-string with a decent sample usage etc etc.
- Add a `tests` folder with the following structure;
```
tests/
__init__.py
test_pypesa.py
```
Note, `tests/__init__.py` can/should be left empty.
- Any other structural improvements that i haven't listed/noticed yet.

So to sum up, i think i can working on this if you think it might be a good idea. Also, i think there other code related improvements suggestion(mostly on the OOD), that i will definitely create their own separate issues.

**Comment by Kalebu** [comment]
Hi Warren  

Thanks for taking  your time putting  all together  

I think its great idea,  you can proceed  with the changes 

+ update the documentation  for every  change  you make 

Let me  incase of anything 


Best regards 
Kalebu
